src/modules/messages/readable.ts

```diff
@@ -14,6 +14,7 @@ export enum AudioCommand {
   AudioMediaStop = 11,
   AudioAlertStart = 12,
   AudioAlertStop = 13,
+  AudioVolumeAdjust = 14
```

Comment:
Unsure whether this is the best idea, I believe we captured all of the commands, but if 14 is used in the future may cause issues. The other option is to perhaps use the default case in the front end to capture messages with no command, open to thoughts?